CHARTER.md

```diff
@@ -1,51 +1,61 @@
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a 66% majority with a quorum of 66% of all ART members is needed for the proposal to pass.
+Each member of the ART is eligible to vote if they have been a member for at least 30 days. For any proposal that impacts the whole project or processes, **approval requires a minimum of four votes with 75% approval among those voting**.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 2 months after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+Any vote must be open for at least 7 calendar days to account for different time zones and other obligations and must be open no longer than 30 days. The voting period must be stated in the proposal in calendar days. The voting period begins on the day the pull request is opened for review and ends at midnight [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth) on the last day of the voting period. Proposals can be extended for no longer than 30 days total. Extensions can be made before the end of the current voting period by the original submitter, their direct representative, or one of the ART chairs by updating the pull request with a note stating the number of calendar days for the extension. **If approval is not reached within the voting period deadline, the proposal is rejected.**
 
-Any voting period must last at least 7 days to account for different time zones and other obligations, and should last no longer than 30 days. The voting period should be stated in the proposal.
+A rejected proposal may only be submitted to the ART again after a miniumum of 30 days following a rejection. If the same proposal is rejected twice, it will not be considered again by the ART.
 
-ART members cast their votes by starting a review that either approves or rejects the decision suggested via a pull request. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the `art-decisions` folder. Silence equals abstaining from a vote.
+ART members cast their votes by reviewing the pull request. An approval vote is cast by approving the pull request, and a rejection vote is cast by using the *Request changes* function and providing a reason for their rejection in the comment. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the [`art-decisions` folder](art-decisions).
 
```

Comment:
This doesn't say _how_ the approved decisions are recorded (i.e., in the [`RECORD.md`](https://github.com/arcalot/arcalot-round-table/blob/main/art-decisions/RECORD.md) file).  (And, it doesn't mention that the PR gets merged into `main`.)

Also, more interestingly, it doesn't say explicitly what happens to rejected decisions.  In point of fact, the PR gets closed without being merged, and that all trace of the proposal, save the PR, is lost...which makes it slightly interesting to enforce the twice-rejected rule, if we retain that.

Should we be recording _all_ decisions, accepted as well as rejected ones?  (After all, rejections **are** _decisions_....)

Reply:
So we would in this case actually merge every proposal, approved or rejected, and the decision either way would always be in the `RECORD.md` file. That's a possible route, admittedly with more administrative overhead. Do we think it is important to have the rejections recorded together with the approvals, or does the PR record suffice for this?

Reply:
The `RECORD.md` needs to be manually maintained already by someone. I do like having a simple list of accepted decisions, but having an `ACCEPTED.md` and `REJECTED.md` might not be a bad idea. I also like *moving* the detailed proposal documents from the "proposal" directory into perhaps an "accepted" or "rejected" directory to keep them organized and more easily searchable.

Reply:
It's already quite a bit of administrative overhead for a chair to create the ratify PR with just an update to the `README.md` file. As the person who has historically handled most of these, I don't want to ask myself to also re-organize files as part of the process.

Reply:
Sure, but it's a trade-off between administrative overhead and clarity.

Note that some of this (including, potentially, the review timeout and default rejection) could be automated by some GitHub Action scripting, if we really wanted to.

Reply:
> The `RECORD.md` needs to be manually maintained already by someone.

@dbutenhof, you mentioned a few applications of GH Actions -- we could conceivably have an Action which records decisions automagically.  (There are lots of possibilities for that....)  And, it could also do things like moving the proposal file to its final resting place.

Note:  I'm marking this conversation as "resolved", because I believe that Dustin has addressed the issues that I raised in the original comment.  If we want to change the currently proposed approach to the details of how decisions are recorded, we should start a new conversation.

Reply:
I would stop short of any changes to the charter with this PR that would require other administrative changes an/or other ART proposals. We could, of course, separately propose and address the automation needs, and then (or concurrently) update the charter accordingly.